airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/base/AirbyteExceptionHandler.java

```diff
@@ -23,11 +51,50 @@ public void uncaughtException(Thread t, Throwable e) {
     // the sync."
     // from the spec:
     // https://docs.google.com/document/d/1ctrj3Yh_GjtQ93aND-WH3ocqGxsmxyC3jfiarrF6NY0/edit#
-    LOGGER.error(logMessage, e);
-    AirbyteTraceMessageUtility.emitSystemErrorTrace(e, logMessage);
+    LOGGER.error(logMessage, throwable);
+
+    // Attempt to deinterpolate the error message before emitting a trace message
+    final String mangledMessage = STRINGS_TO_REMOVE.stream()
+        // Sort the strings longest to shortest, in case any target string is a substring of another
```

Comment:
```suggestion
        // Sort the strings longest to shortest, in case any target string is a substring of another, e.g. "airbyte_intenral" should be swapped out before "airbyte"
```